application/src/navigation/NavBar/NavBar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { PATH } from '../PageRoutes/PageRoutes';
+
+export const NavBar: React.FC = () => (
+  <div>
+    <div>
+      <NavLink to={PATH.CAR}>Car</NavLink>
+    </div>
+    <div>
+      <NavLink to={PATH.CARS}>Cars</NavLink>
+    </div>
+    <div>
+      <NavLink to={PATH.CART}>Cart</NavLink>
+    </div>
+    <div>
+      <NavLink to={PATH.PROFILE}>Profile</NavLink>
+    </div>
+  </div>
```

Comment:
Better for semantics to do navigation:
&lt;nav&gt;
    &lt;ul&gt;
        &lt;li&gt; &lt;NavLink&gt; &lt;/li&gt;
        &lt;li&gt; &lt;NavLink&gt; &lt;/li&gt;
    &lt;/ul&gt;
&lt;/nav&gt;